package/opt/hassbian/suites/upgrade_home-assistant.sh

```diff
@@ -16,6 +16,25 @@ function home-assistant-upgrade-package {
 home-assistant-show-short-info
 home-assistant-show-copyright-info
 
+echo "Checking current version"
+
+function jsonValue() {
+        KEY=$1
+        num=$2
+        awk -F"[,:}]" '{for(i=1;i<=NF;i++){if($i~/'$KEY'\042/){print $(i+1)}}}' | tr -d '"' | sed -n ${num}p
+}
+
+versiongit=$(curl -s -X GET https://api.github.com/repos/home-assistant/home-assistant/releases/latest | jsonValue tag_name 1|sed -e 's/^[[:space:]]*//')
+
+sudo -u homeassistant -H /bin/bash << EOF | grep Version|awk '{print $2'}|while read version; do if [[ ${versiongit} == ${version} ]]; then echo "You already have the latest version: $version";exit 1;fi;done
```

Comment:
This line has the more descriptive error message, but since its in the pipe of the EOF i did it here otherwise you loose the string value.